pyart/graph/radardisplay.py

```diff
@@ -1041,6 +1180,35 @@ def _get_ray_data(self, field, ray, mask_tuple, filter_transitions):
 
         return data
 
+    def _get_azimuth_rhi_data_x_y_z(self, field, target_azimuth,
+                               edges,mask_tuple):
```

Comment:
Just a style issue:
This line is not properly aligned with "self" in the line above and there should be a space after the comma.